pysat/_instrument.py

```diff
@@ -2489,16 +2437,16 @@ def to_netcdf4(self, fname=None, base_instrument=None, epoch_name='Epoch',
         if self._meta_translation_table is None:
             # didn't find a translation table, using the strings
             # attached to the supplied pysat.Instrument object
-            export_name_labels = [self.name_label]
-            export_units_labels = [self.units_label]
-            export_desc_labels = [self.desc_label]
-            export_notes_labels = [self.notes_label]
+            export_name_labels = [self.meta_labels['name'][0]]
+            export_units_labels = [self.meta_labels['units'][0]]
+            export_desc_labels = [self.meta_labels['desc'][0]]
+            export_notes_labels = [self.meta_labels['notes'][0]]
```

Comment:
Does this work?

```suggestion
            export_name_labels = [self.labels.name]
            export_units_labels = [self.labels.units]
            export_desc_labels = [self.labels.desc]
            export_notes_labels = [self.labels.notes]
```

Reply:
Probably?  This would be cleaner, for sure.

Reply:
Or perhaps it would be better to do `self.meta.labels.name`, et cetera.

Reply:
Yeah, that's better style.